index.js

```diff
@@ -89,22 +90,24 @@ module.exports = {
       srcDir: '/',
       destDir: 'velocity'
     });
+    velocityTree = map(velocityTree, 'velocity/velocity.js', function(content) {
+      return 'if (typeof FastBoot === \'undefined\') { ' + content + ' }';
+    });
 
     var matchMediaPath = path.dirname(require.resolve('match-media'));
     var matchMediaTree = new Funnel(this.treeGenerator(matchMediaPath), {
       srcDir: '/',
       destDir: 'match-media'
     });
+    matchMediaTree = map(matchMediaTree, 'match-media/matchMedia.js', function(content) {
+      return 'if (typeof FastBoot === \'undefined\') { ' + content + ' }';
+    });
 
     return mergeTrees([tree, velocityTree, matchMediaTree]);
   },
 
-  included: function(app){
-    if (process.env.EMBER_CLI_FASTBOOT) {
-      // in fastboot we use the shim by itself, which will make
-      // importing velocity a noop.
-      this.import('vendor/shims/velocity.js');
```

Comment:
But as we have no means to prevent importing `velocity/velocity.js` in FastBoot, we either don't need to import the shim (if ember-cli supports AMD imports), or we actually do this already (see https://github.com/ember-animation/liquid-fire/pull/568/files#diff-168726dbe96b3ce427e7fedce31bb0bcL118). So right now there is no difference in how things get imported, regardless of browser or FastBoot environment. Should be ok I think!?